framework/web/UrlManagerInterface.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\web;
+
+use yii\base\InvalidConfigException;
+
+/**
+ * UrlManagerInterface defines an handler for HTTP request parsing and creation of URLs based on a set of rules.
+ *
+ * UrlManager is configured as an application component in [[\yii\base\Application]] by default.
+ * You can access that instance via `Yii::$app->urlManager`.
+ *
+ * @author Nelson J Morais <njmorais@gmail.com>
```

Comment:
API was originally authored by Qiang. I think his authoring copyright should not be completely removed.

```
@author Qiang Xue <qiang.xue@gmail.com>
```

Reply:
@SilverFire done, can you review it?